### PR TITLE
MRC-197 add basic index page

### PR DESCRIPTION
**Pull request by hillalex**
First merge https://github.com/vimc/orderly-web/pull/14 and then merge master into this branch for a clean diff.

This adds an index page to the web app; it's just a placeholder for now to allow us to continue with the auth work.

### CODE REVIEW

**Comment by EmmaLRussell** [comment]
Oh sorry, didn't notice you'd already merged that!

File: src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/TemplateAnotation.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.vaccineimpact.orderlyweb.controllers.api
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/TemplateAnotation.kt]: Filename typo 'Anotation'. Also, shouldn't this be in the 'web' dir under controllers?

Comment (hillalex) [reply src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/TemplateAnotation.kt]: yes it should!

**Comment by hillalex** [comment]
Yeah this is literally just a placeholder so we can keep developing the login, I figured printing a list of report name was as good a placeholder as any!

**Pull request merged**